src/misc/validation/validators/numberValidator.ts

```diff
@@ -1,9 +1,15 @@
+import _ from 'lodash';
```

Comment:
To follow conventions in this project, please import functions directly from the lodash instead of using `_`. I.e `import { identity } from 'lodash'`.